pkg/parse/root.go

```diff
@@ -173,7 +173,7 @@ func (p *root) setSourceStatusWithRetries(ctx context.Context, newStatus sourceS
 
 	currentRS := rs.DeepCopy()
 
-	setSourceStatus(&rs.Status.Source, p, newStatus, denominator)
+	setSourceStatusFields(&rs.Status.Source, p, newStatus, denominator)
```

Comment:
Just curious why do we want to rename the function. Is the intent to make it clear this is different from uppercase one `SetSourceStatus`?

Reply:
I wanted the lowercase name for `setSyncStatus` in run.go, which acts as a wrapper to `SetSyncStatus` that handles updating the state cache, and I wanted to make sure all the setXStatus functions stayed consistently named.

It's kind of a mess that these are all package functions and not struct methods. It makes it hard to really see the layering.

Reply:
If you can think of a better name for the wrapper func, I can rename these back.

Reply:
I see. I missed the new package function `setSyncStatus` defined in another file. It sounds good to have them consistent.